### PR TITLE
Launch control

### PR DESCRIPTION
**Pull request by IanBoyanZhang**
Cap max throttle/accel allowed for onsite test. Add launch control.

### CODE REVIEW

File: ros/src/waypoint_updater/waypoint_updater.py [hunk 1]
```diff
@@ -125,6 +125,8 @@ def distance_sqr(self, x0, y0, x1, y1):
         return (x0 - x1) ** 2.0 + (y0 - y1) ** 2.0
 
     def find_nearest_wp(self, x, y):
+        if getattr(self, 'waypoints') is None:
```
Comment (xperroni) [inline ros/src/waypoint_updater/waypoint_updater.py:128]: This change is unnecessary – `waypoints` is initialized to an empty list in the class constructor, so the attribute will always exist, and never be set to `None`.

**Comment by xperroni** [comment]
The acceleration limit doesn't seem to be enforced – although it's set to `1m/s^2` in the simulation, the car often speeds up by as much as `5m/s^2` when departing from a full stop.

image

**Review by xperroni — CHANGES_REQUESTED** [review]
Please address comments made earlier.

File: ros/src/waypoint_updater/waypoint_updater.py (hunk 1 above)
Comment (IanBoyanZhang) [reply ros/src/waypoint_updater/waypoint_updater.py:128]: Hi Helio, I would have to dive deep into the system before I can give an explanation of root cause. This change was made after `None` exception was observed during onsite test. For simulator solution, this change is unnecessary.

**Comment by IanBoyanZhang** [comment]
`accel_limit` limits throttle value applied to the car. We reduce max acceleration by further limiting max throttle allowed.

**Pull request merged**